pymongo/database.py

```diff
@@ -713,6 +756,36 @@ def _command(
                 client=self.__client,
             )
 
+    @overload
+    def command(
+        self,
+        command: Union[str, MutableMapping[str, Any]],
+        value: Any = 1,
+        check: bool = True,
+        allowable_errors: Optional[Sequence[Union[str, int]]] = None,
+        read_preference: Optional[_ServerMode] = None,
+        codec_options: None = None,
+        session: Optional[ClientSession] = None,
+        comment: Optional[Any] = None,
+        **kwargs: Any,
+    ) -> Mapping[str, Any]:
```

Comment:
Should this be `Dict[str, Any]`? I thought our guiding principle was "accept abstract types as input but return concrete types".